docs/Plan/08. chrome_extension_bookmark_save_plan.md

```
@@ -0,0 +1,1054 @@
+
+**동작 흐름:**
+1. 팝업이 로드되면서 현재 활성화된 탭의 URL과 제목을 읽는다.
+2. 로그인 상태를 확인하고, 로그인이 되어 있다면 백엔드에서 **사용자의 폴더 목록**(`GET /api/folders` 등)을 비동기 조회하여 폴더 선택 드롭다운에 구성한다.
+3. 사용자에게 다음 정보를 담은 크롬 스타일 UI를 노출한다.
+   - **좌측 영역**: 페이지 대표 이미지(썸네일) 또는 파비콘
+   - **우측 영역**: 이름(이름 편집 필드), 폴더(드롭다운, 기본값: 미분류)
+   - **하단 영역**: 상세보기(더보기) 버튼, 완료 버튼, 삭제 버튼
+4. 사용자가 완료를 누르거나 변경 사항을 수정하면 `POST /api/bookmarks`가 호출되어 최종 저장 및 수정이 반영된다.
+
+저장 요청 예시 (기본):
+```json
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)